pages/login.page.js

```diff
@@ -0,0 +1,38 @@
+import BasePage from "./base.page";
+const PAGE_TIMEOUT = 10000;
+const BASE_PAGE = BasePage;
+
+class LoginPage {
```

Comment:
In order to apply inheritance, you need to make `LoginPage` **extend** from `BasePage`. Full class declaration looks like:

```javascript
class LoginPage extends BasePage
```

That way you gain access to superclass (BasePage in this case) attributes and methods by prefixing your attributes / method calls by a **super**.